src/v/redpanda/application.cc

```diff
@@ -461,6 +498,9 @@ void application::wire_up_services() {
 }
 
 void application::start() {
+    _health_ctx.invoke_on_all([](health_context& ctx) { ctx.live = true; })
```

Comment:
wha'ts the different between live and started. isn't live the fact that we responded? do you mean decomissioned or smth else here?

Reply:
On second thoughts, I'm not sure we need these.

Liveness is used to restart a container - we probably don't want that, a `restartPolicy` is probably better.
Readiness is used to remove the container from DNS, but instances are not interchangeable.
Startup is used to bypass issues with liveness probes if startup can take a long time - not needed if there's no liveness probe.